src/ChangeLog.txt

```diff
@@ -2526,11 +2526,11 @@ v0.91.4.8:
     - Possible fix for outpost spawns not being removed from the database when outposts are unclaimed.
     - Add more detail to messages shown in log when Towny enters safe mode. 
 v0.91.4.9:
-    - Remove leading space on the "(PVP)" chunk notification, to match the the no-pvp-message. Thanks for pointing this out Locknlol
+    - Remove leading spaceon the "(PVP)" chunk notification, to match the the no-pvp-message. Thanks for pointing this out Locknlol
     - Fix for servers that do not allow wilderness full build/destroy permissions, where animals could not be killed in the wilderness with arrows.
     - Fix a typo in the CombatUtil, which clarifies which block's destroy permissions we are testing to determine whether animals can be killed.
     - Bump language files to version 0.33   
-    - New Config Option: town.max_claim_radius_value
+    - New Config Optin: town.max_claim_radius_value
```

Comment:
Funky Merge, Removed and Re-Added lines. In the file it still exists. Again, not sure why this happened.